test/built-ins/RegExp/incomplete_hex_unicode_escape.js

```diff
@@ -0,0 +1,16 @@
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
```

Comment:
would you mind adding an `esid: prod-AtomEscape`?

This also helps navigating to the syntax production that predicts the IdentityEscape.

Reply:
That's pretty cool. Is it part of any tooling or is it just there for general convenience?

Reply:
this was initially defined as convenience - identify the spec parts being covered - but it's on the track for tooling (wip). This helps the review process, including we need to look up, investigate and refactor tests in general.